Sources/JXKit/JXContext.swift

```diff
@@ -20,21 +19,24 @@ open class JXContext {
             return scriptLoader.didChange != nil
         }
 
-        /// Whether `require` module support is enabled. Defaults to `true`.
-        public var moduleRequireEnabled: Bool
-        
         /// Configure a global script loader to use as the default when no loader is provided to the `Configuration`.
         ///
         /// - Seealso: ``JXContext/Configuration/scriptLoader``
         public static var defaultScriptLoader: JXScriptLoader = DefaultScriptLoader()
 
         /// The script loader to use for loading JavaScript script files. If the loader vends a non-nil `didChange` listener collection, dynamic reloading will be enabled.
         public var scriptLoader: JXScriptLoader
+
+        /// Configure a global logging function for JX log messages. Defaults to using `print`.
+        public static var defaultLog: (String) -> Void = { print($0) }
+
+        /// The logging function to use for JX log messages.
+        public var log: (String) -> Void
         
-        public init(strict: Bool = true, moduleRequireEnabled: Bool = true, scriptLoader: JXScriptLoader = Self.defaultScriptLoader) {
```

Comment:
I discovered that we don't really need to make require() support configurable. If you ever specify a root URL for resource loading, we add require() so that you can load modules from the file system. Otherwise we don't.